src/reducers/versions.tsx

```diff
@@ -509,6 +513,47 @@ export const getRelativeDiffAnchor = ({
   return null;
 };
 
+export type GetRelativeDiffParams = {
+  _findRelativePathWithDiff?: typeof findRelativePathWithDiff;
+  _getRelativeDiffAnchor?: typeof getRelativeDiffAnchor;
+  currentAnchor: string | void;
+  diff: DiffInfo;
+  pathList: string[];
+  position?: RelativePathPosition;
+  version: Version;
+};
+
+type RelativeDiffResult = {
+  anchor: string | null;
+  path: string | null;
+};
+
+export const getRelativeDiff = ({
+  _findRelativePathWithDiff = findRelativePathWithDiff,
+  _getRelativeDiffAnchor = getRelativeDiffAnchor,
```

Comment:
The coverage report says that no tests are executing `getRelativeDiff()` with the real `getRelativeDiffAnchor()` dependency. We have TS to make sure it should work but it actually wouldn't hurt to add an integration test to cover the real usage of `getRelativeDiffAnchor()` if it's possible to do so.